example-kotlin/contracts/src/main/resources/META-INF/services/io.provenance.scope.contract.contracts.ContractHash

```diff
@@ -0,0 +1 @@
+io.p8e.contracts.examplekotlin.ContractHash1658876514311
```

Comment:
not sure if you wanted to check these in

Reply:
good catch, removed